bin/hummingbot_quickstart.py

```diff
@@ -139,8 +146,48 @@ def main():
         if not login_prompt():
             return
 
-    asyncio.get_event_loop().run_until_complete(quick_start(args))
+    asyncio.get_event_loop().run_until_complete(quick_start(args, docker_pipe, docker_pipe_event))
+
+
+def start_docker(queue, event):
```

Comment:
This code is repeating between the quickstart and the main entry file. It should be refactored out into its own module.